pkg/odo/cli/component/create.go

```diff
@@ -621,7 +621,10 @@ func (co *CreateOptions) Complete(name string, cmd *cobra.Command, args []string
 
 		if co.devfileMetadata.devfileSupport && !co.forceS2i {
 			registrySpinner := log.Spinnerf("Creating a devfile component from registry: %s", co.devfileMetadata.devfileRegistry.Name)
-
+			// warn user of deprecation if registry is git based
```

Comment:
You can probably remove this comment, thecode is explicit enough

Reply:
Removing